### ISSUE TITLE
Try slow double-clicking on a file in the tree view

### ISSUE THREAD

**Issue by revelt** [oriolmirosa/open-files#16]
When I double-click the file in the tree view, the first click adds that file to the "Open Files" list at the top, which instantly pushes down the tree view by one row. Then second click (from my double-click) happens, and it is registered as an intention to open another file, the one above intended file.

It's quite annoying, as occasionally I happen to open a wrong file.

Try slower double-clicking in the tree view. Fast double-click won't cause wrong file opening, but slower will.

Do you see?

I'm wondering maybe a delay would help? Like put a couple seconds delay between the first click on the tree view and "Open Files" item appearing... What do you think?

**Comment by oriolmirosa** [comment]
Thanks for bringing this up. Yes, I also noticed this problem. I followed your suggestion and I'm making some changes to the package to delay the appearance of the file in the 'open files' panel. I made the duration of the delay configurable, and I also added an animation so that that the appearance of the file in the panel is smoother (the duration of the animation is also configurable). I'm dealing with some regressions at the moment and I need to take care of those before I can publish the update, but hopefully I'll get this done within this week. Thanks again for taking the time to report this!
